content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,198 @@
+
+_Learnings:_ Versioning metrics and plots are no less important than data
+versioning.
+
+Countless users asked us when we'd support metrics visualizations. Now it's
+here: DVC 1.0 introduces a metrics file visualization command,
+`dvc metrics diff`. DVC plots are powered by the
+[Vega-Lite](https://vega.github.io/vega-lite/) graphic library.
+
+This function is designed not only for showing visualizations based on the
+current state of your project, but it can also combine multiple plots from your
+Git history in a single chart so you can compare results across commits. Users
+can visualize how, for example, their model accuracy in the latest commit
+differs from another commit (or even multiple commits).
+
+```dvc
+$ dvc plot diff -d logs.csv HEAD HEAD^ d1e4d848 baseline_march
+file:///Users/dmitry/src/plot/logs.html
+$ open logs.html
+```
+
+![](/uploads/images/2020-05-04/dvc-plot.png)
+
+```dvc
+$ dvc plot diff -d logs.csv HEAD HEAD^ d1e4d848 baseline_march \
+        -x loss --template scatter
+file:///Users/dmitry/src/plot/logs.html
+$ open logs.html
+```
+
+![](/uploads/images/2020-05-04/dvc-plot-scatter.png)
+
+### [Data transfer optimizations](https://github.com/iterative/dvc/issues/3488)
+
+_Learnings:_ In ML projects, data transfer optimization is still the king.
+
+We've done substantial work on optimizing data management commands, such as
+`dvc pull \ push \ status -c \ gc -c`. Now, based on the amount of data, DVC can
+choose an optimal data remote traversing strategy.
+
+[Wini-indexes](https://github.com/iterative/dvc/issues/2147) were introduced to
+help DVC instantly check data directories instead of iterating over millions of
+files. This also speeds up file adding\removing to large directories.
+
+More optimizations are included in the release based on performance bottlenecks
+we profiled.
+
+### [Hyperparameter tracking](https://github.com/iterative/dvc/issues/3393)
+
+_Learnings:_ ML pipeline steps depends only on a subset of config file.
+
+This feature was actually released in the last DVC 0.93 version (see
+[params docs](https://dvc.org/doc/command-reference/params). However, it is an
+important step to support configuration files and ML experiments in a more
+holistic way.
+
+I hope our the most active users will find time to check the DVC pre release
+version and provide their feeback. The installation instruction is
+[on our website](https://dvc.org/doc/install/pre-release).
+
+## 5000 GitHub stars
+
+Activity on our GitHub page has grown organically since the DVC repo went public
+on May 4th, 2017. Coincidently, today, in the 3rd year anniversary we have
+reached 5000 starts:
+
+![](/uploads/images/2020-05-04/5k_github.png)
+
+## Thank you!
+
+Thank you again for all DVC contributions, community members, and users. Every
+piece of your help is highly appreciated and will bring huge benefits to the
+entire ecosystem of data and ML projects.
+
+Stay healthy and safe over in your neck of the woods and be in touch on
```

Comment:
`Stay healthy and safe, wherever you are in the world. And be in touch on our Twitter and Discord channel!`